common/eth2_config/src/lib.rs

```diff
@@ -70,6 +70,7 @@ impl Eth2Config {
 pub struct Eth2NetArchiveAndDirectory<'a> {
     pub name: &'a str,
     pub unique_id: &'a str,
```

Comment:
I'm not seeing `unique_id` being used anywhere anymore so I think it can be removed